### PR TITLE
Upgrade Instruction Page and New Blog Post

### PR DESCRIPTION
**Pull request by iamshaynez**
因为在本地做的时候顺序乱了，所以在一条 PR 包含了下面两部分：

**Instruction**

- https://content.docs-68d.pages.dev/upgrade
- https://content.docs-68d.pages.dev/confluence-migration-instructions/

**Blog Post**
- https://content.docs-68d.pages.dev/blog/2024/02/25/2024/sequence-diagram-for-every-architect

### CODE REVIEW

File: src/pages/confluence-migration-instructions.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+![](../../static/img/pages/upgrade-og.jpeg)
+
+## Thank you! We appreciate your choice.
+
+ZenUML Team appreciates your decision of upgrading from our Lite version to Full version. Here's the [Roadmap](https://zenuml.com/roadmap) of our full version product, the new features will be coming soon.
+
+## **Why you need to migrate?**
+
+All the diagrams in your confluence pages created from ZenUML Confluence Lite Version, can not be processed by new installed Full version. The data can not be processed cross different plugins unless the owner of the data(in this case, your organization) apply changes to transmit(we provided migration feature for you to easily do so).
```
Comment (MrCoder) [inline src/pages/confluence-migration-instructions.md:16]: 语法错误：newly created.
> All the diagrams in your Confluence pages, created from ZenUML Confluence Lite Version, cannot be processed by the newly installed Full version.

File: src/pages/confluence-migration-instructions.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+keywords: [ZenUML Freemium, Migration, Upgrade]
+unlisted: false
+---
+
+![](../../static/img/pages/upgrade-og.jpeg)
+
+## Thank you! We appreciate your choice.
+
+ZenUML Team appreciates your decision of upgrading from our Lite version to Full version. Here's the [Roadmap](https://zenuml.com/roadmap) of our full version product, the new features will be coming soon.
+
```
Comment (MrCoder) [inline src/pages/confluence-migration-instructions.md:13]: 添加一段让用户安心的话？
Your data will remain intact
All your diagrams, graphs, and API documents will persist throughout the migration. Even if you accidentally uninstall our application, your data will stay within your Confluence instance, allowing you to regain access simply by reinstalling our application.

**Review by MrCoder — CHANGES_REQUESTED** [review]

File: src/pages/confluence-migration-instructions.md (hunk 1 above)
Comment (iamshaynez) [reply src/pages/confluence-migration-instructions.md:16]: 这里我的本意是新安装的 full version 无法直接操作 lite version create 的 data…

File: src/pages/confluence-migration-instructions.md (hunk 2 above)
Comment (iamshaynez) [reply src/pages/confluence-migration-instructions.md:13]: 好，明天加

File: src/pages/confluence-migration-instructions.md (hunk 1 above)
Comment (iamshaynez) [reply src/pages/confluence-migration-instructions.md]: 我整个大概重写了一下 wording，稍微应该歧义小一些……

**Review by MrCoder — APPROVED** [review]

**Pull request merged**